commit 1be2ee9b89f96bc50d52138c6f12d2979f4715fc
Author: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date:   Fri Jan 31 15:01:28 2025 +0900

    save

diff --git a/src/callbacks.c b/src/callbacks.c
index e06cf5a..57b88b1 100644
--- a/src/callbacks.c
+++ b/src/callbacks.c
@@ -51,10 +51,10 @@ static gboolean RecordAudioSelected(	GtkTreeModel  *model,
 					GtkTreeIter   *iter,
 					gpointer       userdata )
 {
-	gboolean bool;
+	gboolean gbool;
 
-	gtk_tree_model_get( model, iter, AUDIO_COL1_CHECK, &bool, -1);
-	if( bool ) {
+	gtk_tree_model_get( model, iter, AUDIO_COL1_CHECK, &gbool, -1);
+	if( gbool ) {
 		int noPiste;
 
 		gtk_tree_model_get( model, iter, AUDIO_COL8_NO, &noPiste, -1);
@@ -74,11 +74,11 @@ static gboolean RecordSubSelected(	GtkTreeModel  *model,
 					GtkTreeIter   *iter,
 					gpointer       userdata )
 {
-	gboolean bool;
+	gboolean gbool;
 
-	gtk_tree_model_get( model, iter, SUB_COL1_CHECK, &bool, -1);
+	gtk_tree_model_get( model, iter, SUB_COL1_CHECK, &gbool, -1);
 
-	if( bool ) {
+	if( gbool ) {
 		int noPiste;
 
 		gtk_tree_model_get( model, iter, SUB_COL6_NO, &noPiste, -1);
